CMakeLists.txt

```diff
@@ -102,6 +102,14 @@ if(HAVE_X11)
     target_link_libraries(axpbox X11)
 endif()
 
+if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O0 -ggdb")
```

Comment:
There is a specific optimization flag for debugging (`-Og`) which can be used instead of `-O0`.